src/dyn/dyn_variables.cpp

```diff
@@ -615,7 +615,7 @@ vector< vector<double> > dyn_variables::get_fssh3_errors(){
 vector<double> dyn_variables::get_fssh3_average_errors(){
   int itraj, k;
   vector<double> res(5,0.0);
-  for(itraj;itraj<ntraj;itraj++){
+  for(itraj=0;itraj<ntraj;itraj++){

```

Comment:
interesting - how did the older version even worked? ))

Reply:
Default value is zero

Reply:
potentially it could have been some trash value, so definitely not a safe implementation; i guess the compilers are smart enough to set it to zero